src/item.cpp

```diff
@@ -2237,12 +2264,16 @@ bool item::is_auxiliary_gunmod() const
 
 int item::get_storage() const
 {
-    const auto t = find_armor_data();
+    float pockets = 1;
+    auto t = find_armor_data();
     if( t == nullptr ) {
         return 0;
     }
+    if (pocketed == true){
+        pockets = (1.5); //Test value
+    }
     // it_armor::storage is unsigned char
-    return static_cast<int>( static_cast<unsigned int>( t->storage ) );
+    return (static_cast<int> (static_cast<unsigned int>( t->storage * pockets) ) );
```

Comment:
`(t-> storage > 0 ? t-> storage * pockets : t-> storage + pockets)`
Functionally equivalent, unless I've forgot the right order *again*, to "if (t-> storage > 0) { t-> storage * pockets; } else { t->storage + pockets }. Would solve the issue of non-pocketed items.

And applying one to cloak would make you look like a kender.